imports/plugins/core/checkout/client/templates/checkout/payment/methods/cards.js

```diff
@@ -42,3 +45,89 @@ Template.corePaymentMethods.helpers({
     return self;
   }
 });
+Template.payWithWallet.onCreated(function bodyOnCreated() {
+  this.state = new ReactiveDict();
+  this.state.set("details", { balance: 0 });
+  this.autorun(() => {
+    this.subscribe("transactionInfo", Meteor.userId());
+    const transactionDetail = Wallet.find().fetch();
+    this.state.set("details", transactionDetail[0]);
+  });
+});
+Template.payWithWallet.events({
```

Comment:
**Code Quality - Critical**
Please, drastically refactor this template. Resolve with functional approach